appengine/flexible/sparkpost/main_test.py

```diff
@@ -0,0 +1,55 @@
+import pytest
```

Comment:
need instructions for running tests. 

none of the methods showed in the pytest doc worked.